.repo-metadata.json

```diff
@@ -3,11 +3,12 @@
     "name_pretty": "Google Cloud Common",
     "client_documentation": "https://cloud.google.com/python/docs/reference/common/latest",
     "issue_tracker": "",
-    "release_level": "ga",
+    "release_level": "stable",
     "language": "python",
     "library_type": "GAPIC_AUTO",
     "repo": "googleapis/python-cloud-common",
     "distribution_name": "google-cloud-common",
     "default_version": "",
-    "codeowner_team": ""
+    "codeowner_team": "",
+    "api_shortname": "common"
```

Comment:
This library only has message types, is the `api_shortname` still required?

Reply:
Fixed.